### ISSUE TITLE
querying for gene expression without selecting a region causes machine to freeze

### ISSUE THREAD

**Issue by xgui3783** [FZJ-INM1-BDA/siibra-python#41]
following from #38 

to reproduce:

```python
import siibra as sb
atlas=sb.atlases.MULTILEVEL_HUMAN_ATLAS
# deliberately not selecting a region
# atlas.select_region("V1")
features = atlas.get_features(
    siibra.modalities.GeneExpression, 
    gene=siibra.features.gene_names.MAOA)
```

expected result: raise exception (need to select region), or use a more efficient way to producing the mask

actual result: tries to calculate masks for every sub regions, and eventually freezes/OOMs

**Comment by dickscheid** [comment]
Oh my god, this was really bad. Region.build_mask() recursed into children, although it is not necessary for labelled maps: We can collect the labels of all children on the object (in fact already stored in Region.labels), and then use np.isin to mask those pixels right away without recursing into child regions. 

Fixed by f70a5f1.

**Comment by dickscheid** [comment]
(now, the same query above runs in a fraction of a second on my macbook)
